lint.sh

```diff
@@ -1,17 +1,32 @@
 #!/bin/bash
 
-cd "${GITHUB_WORKSPACE}" || exit
+cd "${GITHUB_WORKSPACE}" || exit 1
+
+# https://github.com/reviewdog/reviewdog/issues/1158
+git config --global --add safe.directory "$GITHUB_WORKSPACE" || exit 1
 
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-clj-kondo --lint $(find "${INPUT_PATH}" -not -path "${INPUT_EXCLUDE}" -type f -name "${INPUT_PATTERN}") \
+sources=$(find "${INPUT_PATH}" -not -path "${INPUT_EXCLUDE}" -type f -name "${INPUT_PATTERN}")
+
+echo "::group::Files to lint"
+echo "${sources}"
+echo "::endgroup::"
+
+clj -Sdeps '{:deps {clj-kondo/clj-kondo {:mvn/version "RELEASE"}}}' -M -m clj-kondo.main \
```

Comment:
This is what keeps the kondo version current?

Reply:
It's just using the clojure CLI to pull the latest clj-kondo on invocation of this line, rather than loading the version in the docker image- which wasn't always up-to-date and also had the potential to be cached

Reply:
👏